plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationConnectorSmokeTest.java

```diff
@@ -523,12 +598,83 @@ protected QueryRunner createQueryRunner()
         kafka.sendMessages(duplicateValuesInColumnsRecordsBuilder.build().stream(), schemaRegistryAwareProducer(kafka));
         pinot.createSchema(getClass().getClassLoader().getResourceAsStream("duplicate_values_in_columns_schema.json"), DUPLICATE_VALUES_IN_COLUMNS_TABLE);
         pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("duplicate_values_in_columns_realtimeSpec.json"), DUPLICATE_VALUES_IN_COLUMNS_TABLE);
+    }
 
-        return PinotQueryRunner.createPinotQueryRunner(
-                ImmutableMap.of(),
-                pinotProperties(pinot),
-                Optional.of(binder -> newOptionalBinder(binder, PinotHostMapper.class).setBinding()
-                        .toInstance(new TestingPinotHostMapper(pinot.getBrokerHostAndPort(), pinot.getServerHostAndPort(), pinot.getServerGrpcHostAndPort()))));
+    private void createAndPopulateNationAndRegionData(TestingKafka kafka, TestingPinotCluster pinot)
```

Comment:
Could you load nation and region table from `tpch.tiny` schema? Thus, we don't need nation_data.txt file. You can use `computeActual` method.

Reply:
The queries from `BaseConnectorSmokeTest` check for `pinot.default.nation` and `pinot.default.region`. Can you maybe point me to an example on the `tpch.tiny` schema? I tried searching and it seemed to be like the database changes from `pinot.default` to `tpch.tiny`

Reply:
Sorry for my poor explanation. I meant loading rows from `tpch.tiny.nation` & `tpch.tiny.region` using `SELECT` statement.

Reply:
No worries, thanks. Will check this.

Reply:
I tried to use the `queryRunner().execute` to see if I can use the existing schema to fetch the data, something like this: `MaterializedResult materializedRows = queryRunner.execute("Select * from tpch.tiny.region");`, but apparently this table gets queried in `createPinotQueryRunner`. 

I cannot realise how the library, `tpch`, can include this data into a runner for reuse. Any pointers you can show if I am missing something?

Reply:
Could you add below lines in `PinotQueryRunner`? 
```java
        queryRunner.installPlugin(new TpchPlugin());
        queryRunner.createCatalog("tpch", "tpch");
```